elixir/apps/api/lib/api/gateway/channel.ex

```diff
@@ -130,16 +136,43 @@ defmodule API.Gateway.Channel do
     end
   end
 
-  def handle_info({:reject_access, client_id, resource_id}, socket) do
+  # Resource is updated, eg. traffic filters are changed
+  def handle_info({:update_resource, resource_id}, socket) do
+    OpenTelemetry.Ctx.attach(socket.assigns.opentelemetry_ctx)
+    OpenTelemetry.Tracer.set_current_span(socket.assigns.opentelemetry_span_ctx)
+
+    OpenTelemetry.Tracer.with_span "gateway.resource_updated",
+      attributes: %{resource_id: resource_id} do
+      resource = Resources.fetch_resource_by_id!(resource_id)
+      push(socket, "resource_updated", Views.Resource.render(resource))
```

Comment:
@conectado gateway can receive `resource_updated` message, it can be used to update traffic filters once they are implemented, but you can just ignore it right now (but make sure to decode it so we don't crash).